File: profiles/mo6tw/hud/saveicon.js
root.SaveIcon = {
    ForegroundAnimation: "SaveIcon",
    DefaultPosition: { X: 1072, Y: 36 },
    BackgroundSprite: "SaveIconBg",
    BackgroundOffset: { X: -7, Y: -4 },
    BackgroundMaxAlpha: 0.5,
    FadeInDuration: 0.25,
    FadeOutDuration: 0.25
};

MakeAnimation({
    Name: "SaveIcon",
    Sheet: "Data",
    FirstFrameX: 257,
    FirstFrameY: 121,
    FrameWidth: 96,
    ColWidth: 96,
    FrameHeight: 22,
    RowHeight: 24,
    Frames: 4,
    Duration: 0.4,
    Rows: 4,
    Columns: 1,
    PrimaryDirection: AnimationDirections.Down
});

root.Sprites["SaveIconBg"] = {
    Sheet: "Data",
    Bounds: { X: 1439, Y: 1, Width: 84, Height: 84 }
};

File: profiles/mo8/hud/saveicon.lua
root.SaveIcon = {
    ForegroundAnimation = "SaveIcon",
    DefaultPosition = { X = 1672, Y = 32 },
    BackgroundSprite = "SaveIconBg",
    BackgroundOffset = { X = -10.5, Y = -6 },
    BackgroundMaxAlpha = 0.5,
    FadeInDuration = 0.5,
    FadeOutDuration = 0.25
};

MakeAnimation({
    Name = "SaveIcon",
    Sheet = "Data",
    FirstFrameX = 312,
    FirstFrameY = 634,
    FrameWidth = 216,
    ColWidth = 216,
    FrameHeight = 56,
    RowHeight = 64,
    Frames = 4,
    Duration = 0.7,
    Rows = 4,
    Columns = 1,
    PrimaryDirection = AnimationDirections.Down
});

root.Sprites["SaveIconBg"] = {
    Sheet = "Data",
    Bounds = { X = 2158.5, Y = 1.5, Width = 126, Height = 126 }
};